Update prometheus configuration

glo51 opened the pull request.

glo51 committed:
Change ContainerCPUUsageHigh rule threshold from 80% to 95%

glo51 committed:
Update prometheus template with k8s token file

glo51 committed:
Configure Prometheus kubeconfig template

glo51 committed:
Remove 'instance=~"de-.*"' requirement from "HighCpuLoad above 95 percent" alert

glo51 committed:
Add tasks to load admin kubeconfig and create Prometheus kubeconfig based on it

tomasz-baran commented on ansible/playbooks/roles/prometheus/tasks/configure-k8s-apps-monitoring.yml:
```diff
@@ -32,6 +24,27 @@
     mode: u=r,go=
   no_log: true
 
+- name: Load admin kubeconfig file into Ansible
```
```suggestion
- name: Load admin kubeconfig file
```

tomasz-baran commented on ansible/playbooks/roles/prometheus/tasks/configure-k8s-apps-monitoring.yml:
```diff
@@ -32,6 +24,27 @@
+    src: "{{ kubeconfig.local }}"
+  register: admin_kubeconfig_file
+
+- name: Get kubeconfig structure from file
```
```suggestion
- name: Parse kubeconfig file
```

glo51 committed:
Update ansible/playbooks/roles/prometheus/tasks/configure-k8s-apps-monitoring.yml

Remove "into Ansible" from task "Load admin kubeconfig file into Ansible"

Co-authored-by: Tomasz Baran <110602076+tomasz-baran@users.noreply.github.com>

glo51 committed:
Update ansible/playbooks/roles/prometheus/tasks/configure-k8s-apps-monitoring.yml

Update task name from "Get kubeconfig structure from file" to "Parse kubeconfig file"

Co-authored-by: Tomasz Baran <110602076+tomasz-baran@users.noreply.github.com>

glo51 committed:
Remobe bearer token file from scrapers: node-exporter and file_service_discovery

glo51 committed:
Move "bearer_token_file: k8s-token" lines after "job_name:" lines

tomasz-baran reviewed (commented):
Please update docs/changelogs/CHANGELOG-2.1-PGML.md

glo51 committed:
Update changelog
